stable/ghost/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 name: ghost
-version: 1.0.2
+version: 2.0.0
```

Comment:
This should be a minor version update and not major.

Reply:
please ignore this comment as we agreed that major version update is the right thing